backend/service/aws/iam.go

```diff
@@ -0,0 +1,16 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+)
+
+func (c *client) SimulateCustomPolicy(ctx context.Context, region string, customPolicySimulatorParams *iam.SimulateCustomPolicyInput) (*iam.SimulateCustomPolicyOutput, error) {
+	rc, err := c.getRegionalClient(region)
```

Comment:
just curious, `region` is not used in the AWS IAM client, afaik. But I see this is a general field for clutch clients, is it just ignored internally?

Reply:
It is used internally, when configuring a client with iam.NewFromConfig(regionCfg). https://github.com/lyft/clutch/blob/main/backend/service/aws/aws.go#L94-L100

We set the client to use a region. When we consume the API we can call it like this: ```c.awsClient.SimulateCustomPolicy(...,reigon,...)``` and it will select the client with the reigon already configured. 

See: 
https://github.com/lyft/clutch/blob/main/backend/service/aws/aws.go#L71
https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/

The reason this is done is because some services are only available in one location, and the fact that it helps a bit with latency when doing AWS operations.

Reply:
The python library ([boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#client)) obviates the region field, so I wondered if this would do the same.